website/docs/sql_queries.md

```diff
@@ -362,37 +349,37 @@ Following tables show whether a given query is supported on specific query engin
-| **Impala**            |Y|N|
-| **Redshift Spectrum** |Y|N|
-| **Doris**             |Y|N|
-| **StarRocks**         |Y|N|
-| **ClickHouse**        |Y|N|
+| Query Engine          | Snapshot Queries | Incremental Queries |
+|-----------------------|------------------|---------------------|
+| **Hive**              | Y                | Y                   |
+| **Spark SQL**         | Y                | Y                   |
+| **Flink SQL**         | Y                | N                   |
+| **PrestoDB**          | Y                | N                   |
+| **Trino**             | Y                | N                   |
+| **AWS Athena**        | Y                | N                   |
+| **BigQuery**          | Y                | N                   |
+| **Impala**            | Y                | N                   |
+| **Redshift Spectrum** | Y                | N                   |
+| **Doris**             | Y                | N                   |
+| **StarRocks**         | Y                | Y                   |
+| **ClickHouse**        | Y                | N                   |
 
 ### Merge-On-Read tables
 
-| Query Engine        |Snapshot Queries|Incremental Queries|Read Optimized Queries|
-|---------------------|--------|-----------|--------------|
-| **Hive**            |Y|Y|Y|
-| **Spark SQL**       |Y|Y|Y|
-| **Spark Datasource** |Y|Y|Y|
-| **Flink SQL**       |Y|Y|Y|
-| **PrestoDB**        |Y|N|Y|
-| **AWS Athena**      |Y|N|Y|
-| **Big Query**       |Y|N|Y|
-| **Trino**           |N|N|Y|
-| **Impala**          |N|N|Y|
-| **Redshift Spectrum** |N|N|N|
-| **Doris**           |N|N|N|
-| **StarRocks**       |N|N|N|
-| **ClickHouse**      |N|N|N|
+| Query Engine          | Snapshot Queries | Incremental Queries | Read Optimized Queries |
+|-----------------------|------------------|---------------------|------------------------|
+| **Hive**              | Y                | Y                   | Y                      |
+| **Spark SQL**         | Y                | Y                   | Y                      |
+| **Spark Datasource**  | Y                | Y                   | Y                      |
+| **Flink SQL**         | Y                | Y                   | Y                      |
+| **PrestoDB**          | Y                | N                   | Y                      |
+| **AWS Athena**        | Y                | N                   | Y                      |
+| **Big Query**         | Y                | N                   | Y                      |
+| **Trino**             | N                | N                   | Y                      |
+| **Impala**            | N                | N                   | Y                      |
+| **Redshift Spectrum** | N                | N                   | Y                      |
+| **Doris**             | N                | N                   | N                      |
+| **StarRocks**         | Y                | Y                   | Y                      |
```

Comment:
Incremental queries are not supported correct ?

Reply:
No, I ran the example mentioned [here](https://github.com/StarRocks/demo/tree/master/documentation-samples/hudi) incremental queries are not supported in both MOR and COW.